auto/modules/wasm

```diff
@@ -0,0 +1,207 @@
+# Copyright (C) Andrew Clayton
+# Copyright (C) F5, Inc.
+
+
+NXT_WASM_RUNTIME=wasmtime
+
+shift
+
+for nxt_option; do
+
+    case "$nxt_option" in
+        -*=*) value=`echo "$nxt_option" | sed -e 's/[-_a-zA-Z0-9]*=//'`     ;;
```

Comment:
Isn't it better to use POSIX's `[[:alnum:]]` for sed(1)?  Or maybe that's not portable to some old or weird systems?